src/Controller/ListPageRssController.php

```diff
@@ -196,19 +208,19 @@ public function build(NodeInterface $node): CacheableResponse {
     ])->toString(TRUE);
     $atom_link = $this->request->getCurrentRequest()->getSchemeAndHttpHost() . $this->request->getCurrentRequest()->getRequestUri();
     $cache_metadata->addCacheableDependency($default_link);
-    $default_description = $this->getChannelDescription($node, $cache_metadata);
     $build = [
       '#theme' => 'oe_list_pages_rss',
       '#title' => $default_title,
       '#link' => $default_link->getGeneratedUrl(),
       '#atom_link' => $atom_link,
-      '#channel_description' => empty($default_description) ? $default_title : $default_description,
       '#language' => $language->getId(),
       '#copyright' => $this->getChannelCopyright(),
       '#image' => $this->getChannelImage($default_link, $cache_metadata),
       '#channel_elements' => [],
       '#items' => $this->getItemList($node, $cache_metadata),
     ];
+    $default_description = $this->getChannelDescription($node, $cache_metadata);
+    $build['#channel_description'] = empty($default_description) ? $default_title : $default_description;
```

Comment:
Moved this below so the list gets executed before trying to get the description where we can use the results from the execution.